react/internal/http-switch-link.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file defines a linker middleware used for changing http verbs, allowing
+ * cache in CDNs and other http-based caching content delivery networks
+ *
+ * If the query type is Query, the request will be made using an HTTP GET
+ * If the query type is Mutation, the request will be made using HTTP POST
+ *
+ */
+import {ApolloLink} from 'apollo-link'
+import { prop, map, contains, isEmpty, isNil } from 'ramda'
```

Comment:
We shouldn't use ramda in the runtime as it will add to bundle size. Incidentally, you were able to use it without declaring because we have a node modules leak in the builder hub. Need to look into that.

Reply:
Ramda removed. Had to do some ugly code though :(